backend/tracim_backend/lib/utils/authentification.py

```diff
@@ -143,9 +148,35 @@ def get_current_user(self, request: TracimRequest) -> typing.Optional[User]:
             user = self._get_user_api(request).get_one(user_id=request.unauthenticated_userid)
         except UserDoesNotExist:
             user = None
-        # do not allow invalid_user + ask for cleanup of session cookie
+        # do not allow invalid_user
         if not user or not user.is_active or user.is_deleted:
             return None
+
+        # ensure cookie expiry date is updated if it is too old
```

Comment:
Follow coding rule for comment here : 
```suggestion
        # INFO - SG - 2020-08-13  ensure cookie expiry date is updated if it is too old
```

Reply:
I feel like we should re-discuss this rule. It seems a bit overkill in most cases.

Reply:
I agree: for "simple" comments it seems too much to me. I would reserve the `INFO` when explaining a "strange"/"unexpected" behavior.